### PR TITLE
R script corrections aws

### PR DESCRIPTION
**Pull request by evanbiederstedt**
This should take a single TN pair, with no paths to the BAMs. 

Note: the working directory equals the output directory

```
Rscript /data/LOHHLAscript.R  --patientId 'someStringHere'  --normalBAMfile SB803157-N.bam   

   --tumorBAMfile EE669269-T.bam  --HLAfastaLoc /data/abc_complete.fasta  

   --HLAexonLoc /lohhla/data/hla.dat     --hlaPath /data/massage.hla.txt   

   --gatkDir /picard     --novoDir /novocraft     --outputDir /data/test_LOHHLA/ 

   --CopyNumLoc /data/hey.txt 
```

Try via 

```
docker run -it -v /Users/biederse/downloads:/data  cmopipeline/lohhla:0.9.3
```

### CODE REVIEW

**Comment by kpjonsson** [comment]
Testing.

**Comment by evanbiederstedt** [comment]
@kpjonsson I forget to mention

```
$ cat hey.txt
tumorPurity	tumorPloidy
0.5778085	3.522477
```

and

```$ cat massage.hla.txt  
hla_a_03_01_01_01
hla_a_11_01_01
hla_b_07_02_01
hla_b_55_01_01
hla_c_03_03_01
hla_c_07_02_01_03
```

File: LOHHLAscript.R [hunk 1]
```diff
@@ -578,26 +671,27 @@ if(mapping.step){
     regionDir <- paste(workDir, '/', BAMid, sep = '')
```
Comment (kpjonsson) [inline LOHHLAscript.R]: Seems to only work when input files are in the run directory due to how this is defined.

**Comment by evanbiederstedt** [comment]
@kpjonsson 
(I'm confused that I can edit your comments)

> Seems to only work when input files are in the run directory due to how this is defined.

Yep, I punted. I believe the original sin is here (my note):

`workDir           <- opt$outputDir     ### NOTE: outputDir is the workDir`

The reason this was done is that the R script is a wrapper around standard bioinformatic tools, and he wanted to access the intermediate output files.

**Comment by evanbiederstedt** [comment]
Last question, @kpjonsson 

Could you detail what you tested this on?

**Comment by kpjonsson** [comment]
I reran an exome sample I had run before, the results came back identical. I can share details on Slack if you want.

**Comment by evanbiederstedt** [comment]
Sounds good to me; I'll trust @kpjonsson

**Pull request merged**